datadog_checks_base/datadog_checks/base/utils/db/statement_metrics.py

```diff
@@ -104,6 +105,33 @@ def compute_derivative_rows(self, rows, metrics, key):
         return result
 
 
+def merge_duplicate_rows(rows, metrics, key):
+    """
+    Given a list of query rows, apply query normalization and compute query signatures to detect duplicate
+    queries_by_key and merge them into a single row.
```

Comment:
nit: comment is now out of date

Also might be good to provide some context for the bug and why this exists.

Reply:
Thanks. Updated comment.